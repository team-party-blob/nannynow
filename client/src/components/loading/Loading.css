#loadingText {
  font-size: 35px;
}