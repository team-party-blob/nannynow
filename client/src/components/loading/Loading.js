import React, { Fragment } from 'react';
import styles from './Loading.css';

export default function Loading() {
  return (
    <Fragment>
      <h1 id={styles.loadingText}>Loading</h1>
    </Fragment>
  );
}
