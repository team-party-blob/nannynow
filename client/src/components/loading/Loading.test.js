import React from 'react';
import { shallow } from 'enzyme';
import Loading from './Loading';

jest.mock('../../assets/nannyNowLogo.png');

describe('Loading component', () => {
  it('renders a snapshot component', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
