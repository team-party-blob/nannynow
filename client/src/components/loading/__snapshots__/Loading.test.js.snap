// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders a snapshot component 1`] = `
<Fragment>
  <h1
    id="loadingText"
  >
    Loading
  </h1>
</Fragment>
`;
